docs/content/en/content-management/multilingual.md

```diff
@@ -47,6 +47,13 @@ weight = 2
 linkedin = "https://linkedin.com/fr/whoever"
 [languages.fr.params.navigation]
 help  = "Aide"
+
+[languages.ar]
+title = "مدونتي"
+weight = 2
+languagedirection = "rtl"
+[languages.ar.params]
+linkedin = "https://linkedin.com/ar/whoever" 
```

Comment:
Drop that param (we use example.com for all example links, and that would not work here, so just drop it).